src/js/App.js

```diff
@@ -0,0 +1,29 @@
+import "../css/index.css";
+
+import { Main } from "./components";
+import { Header } from "./layout";
+
+import { $createElement } from "./utils/dom";
```

Comment:
```suggestion
import { $createElement } from "./utils";
```
utils 폴더에도 index.js를 하나 만들어주면 좋겠네요!?